website/content/v1.2/_index.md

```diff
@@ -4,9 +4,9 @@ no_list: true
 linkTitle: "Documentation"
 cascade:
   type: docs
-lastRelease: v1.2.5
-kubernetesRelease: "1.25.1"
-prevKubernetesRelease: "1.24.3"
```

Comment:
this can't be true, it's v1.1 Kubernetes version

Reply:
ah I think i looked at the wrong release, fixing :facepalm: